spyder_kernels/console/kernel.py

```diff
@@ -507,3 +508,12 @@ def _load_wurlitzer(self):
                 get_ipython().run_line_magic('reload_ext', 'wurlitzer')
             except Exception:
                 pass
+    def set_equations_color(self, background_color):
```

Comment:
@ccordoba12 maybe we should define a default?

Also I think we should use, light/dark (I saw the example @juanis2112 used was white vs dark?